packages/web/src/components/edit/fields/visibility/VisibilityField.tsx

```diff
@@ -76,6 +84,13 @@ export const VisibilityField = (props: VisibilityFieldProps) => {
     ? 'hidden'
     : 'public'
 
+  const parentFormIsInitiallyUnlisted =
```

Comment:
could we not just use the "isHidden" value and pair that with "!isUpload"?

Reply:
oh, we need the initial value, but forgot we could just get that from the useField results

update inc

Reply:
yaya exactly, thanks :)